main.py

```diff
@@ -93,3 +93,16 @@ def add_nosniff_content_type_header(response):
     response.headers["X-Content-Type-Options"] = "nosniff"
     response.headers["X-Frame-Options"] = "deny"
     return response
+
+@app.after_request
```

Comment:
(asking not knowing a lot about flask) -- it seems weird to add a header after the response is sent. Or does `after_request` actually run before the response is sent?

Reply:
found the answer. it's run after the request is processed but before the response is sent...
![flask request response timing](https://instructobit.com/images/e46ed22c-5638-4b57-ac6d-c8388b720319/Python-Flask-request-processing-lifecycle.jpg)
https://instructobit.com/tutorial/111/Python-Flask%3A-running-code-before-and-after-every-request